netlify/netlify-plugin-nextjs
# fix: add missing middleware runtime file

### Summary
Adds a missing file to the function bundle, required by middleware in Next 12

### Test plan

1. Visit [the Deploy Preview at `/middle/`](https://deploy-preview-762--netlify-plugin-nextjs-demo.netlify.app/middle)
2. Observe the message "hello middleware"

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal (small picture)

Fixes #760 

![image](https://user-images.githubusercontent.com/213306/140071909-adf5599f-a2a2-48e8-96f1-b9775a7761eb.png)
